tools/cli/commands.js

```diff
@@ -1767,6 +1767,7 @@ main.registerCommand({
   maxArgs: 1,
   options: {
     db: { type: Boolean },
+    'skip-cache': { type: Boolean },
```

Comment:
Have we documented this option for `skip-cache`?

Reply:
I thought about it, but it's mainly for internal use. The profile script prevents repeated clearing of the performance repository context. Cache clearing always happens with `meteor reset`, and I can't see a case where a dev wouldn't want that when used intentionally. Many commands in this file have undocumented options for internal use.